legacy-sdk/whirlpool/tests/utils/init-utils.ts

```diff
@@ -156,10 +156,15 @@ export async function buildTestAquariums(
 
   const aquariums: TestAquarium[] = [];
   // Airdrop SOL into provider wallet;
-  await ctx.connection.requestAirdrop(
+  const balance = await ctx.connection.getBalance(
     ctx.provider.wallet.publicKey,
-    100_000_000_000_000,
   );
+  if (balance < 10_000_000_000) {
+    await ctx.connection.requestAirdrop(
+      ctx.provider.wallet.publicKey,
+      50_000_000_000, // 50 SOL
+    );
+  }
   for (const initParam of initParams) {
     // Create configs
     let configParams = initParam.configParams;
```

Comment:
i think smth like this would let us get rid of the shebang operator throughout the tests

```ts
const configParams =
  initParam.configParams ?? generateDefaultConfigParams(ctx);
```